dev-python/pyparsing/pyparsing-2.4.5.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} pypy pypy3 )
```

Comment:
I had tons of error trying to merge this, only to realize my previous version and its revdeps had python3.8 already. Also pypy has been removed from this ebuild. 

I'm gonna have to hold this, as it's not maintained by me or any of the projects I'm involved in. You **should** open a bug in bugzilla, then link this PR to that bug, and **then** I can step in after 14-30 days if nothing is heard from the actual maintainer. 
per latest gentoo-project ML discussions.
https://devmanual.gentoo.org/general-concepts/package-maintainers/#maintainer-authority

Reply:
https://bugs.gentoo.org/704842